config/graphql.yml

```
@@ -0,0 +1,9 @@
+require: rubocop-graphql
+
+GraphQL/ExtractInputType:
+  Enabled: false
```

just add a empty line

```suggestion
  Enabled: false
  
```